### ISSUE TITLE
Creditcard should not be base component

### ISSUE THREAD

**Issue by Jacktse1983** [compareasiagroup/cgg-core-js#2]
cgg-creditcard
This is more looks like a collection of other base elements, or we create folder
/elements - a collection of base elements to form components
/components - formed by basic elements, has self UI and function

**Comment by Ramon2000** [comment]
Thinking about splitting it up in:
1) CGG-creditcard-header
2) CGG-creditcard-body
3) CGG-creditcard-footer
4) CGG-creditcard-fold-out

**Comment by Jacktse1983** [comment]
To me with verticals it should be in second layer, we still need the library for PL, so this should not be in library IMHO

**Comment by Ramon2000** [comment]
Well we can replace creditcard with simply product.
Because the creditcard and personalloan single entry product is the same.

**Comment by Jacktse1983** [comment]
OK, then I guess we can rename it as product_item etc.
As a library it should isolated from the UI design as much as possible.

We need to keep the folder structure clear for future maintenance. If some CR wants us to change the UI we need to go to library and do that again, it is OK for one country, what about other countries use the same library? We don't want to use different versions for different countries just because of localization do we?

**Comment by Ramon2000** [comment]
Yes, we have to look again at the names. The point is valid and will incorporate it from now on.
